FirstAnalysis/Correlations/doTemplate.C

```diff
@@ -0,0 +1,398 @@
+///////////////////////////////////////////////////////////////////////////
+//  Macro to perform a template fit on 1D histograms of projected correlations on delta phi
+//
+//  This macro allows you to fit the correlation vs. delta phi and extract V_nDelta coefficients
+//  for both reference and differential flow.
+//  v_n = sqrt{V_nDelta} -> this is done in the final macro getFlow.C
+//
+//  Input: file with histograms produced by doPhiProjections.C
+//
+//  Usage: root -l doTemplate.C
+//
+//  Parameters:
+//  - inputFileName: input file
+//  - outputFileName: output file containing V_nDelta results
+//  - outputPlotsName: name of the folder to store plots
+//  - drawSeparatevn: flag to draw the template fit result for v2 and v3 separately instead of sum of vn
+//  - drawTemplate: flag to draw the result of the template fit with the correlation
+//  - savePlots: flag to save drawn plots
+//
+//  Contributors:
+//    Katarina Krizkova Gajdosova <katarina.gajdosova@cern.ch>
+//    Gian Michele Innocenti <gian.michele.innocenti@cern.ch>
+///////////////////////////////////////////////////////////////////////////
+
+int nBinspTtrig = 6;
+double binspTtrig[] = {0.2, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0};
+
+int nBinspTref = 1;
+double binspTref[] = {0.2, 3.0};
+
+const int nBinsMult = 7;
+double binsMult[] = {0, 10, 20, 30, 40, 50, 60, 80, 100, 200};
+
+TFile* outputFile;
+
+TH1D *hminuit, *hminuit_periph;
+
+TH1D* hDifferentialV2[nBinsMult];
+
+void tempminuit(double* fParamVal, double* fParamErr);
+void minuitfcn(int& npar, double* gin, double& ff, double* par, int iflag);
+double fun_template(double* x, double* par);
```

Comment:
Please unify the names of functions to follow the same style (ideally camelCase).

Reply:
Done, I hope it is good now :)